app/templates/content/index.hbs

```diff
@@ -44,6 +44,9 @@
             </div>
         </div> {{!END CONTAINER DIV}}
     </div> {{!END HEADER ROW}}
+    {{#if isAdmin}}
```

Comment:
I agree that moderators should be able to get to the moderation view from the preprint detail view or search results. I added a [Jira ticket](https://openscience.atlassian.net/browse/MOD-64) for post-MVP.